pkg/webhookhandler/validator_test.go

```diff
@@ -173,13 +173,15 @@ func newRequest(operation admissionv1.Operation, oldObj, newObj *v1alpha1.Kubevi
 		},
 	}
 
+	var defaultReqDryRun bool = false
 	switch operation {
 	case admissionv1.Create:
 		req.Object = runtime.RawExtension{
 			Raw:    []byte(runtime.EncodeOrDie(encoder, oldObj)),
 			Object: oldObj,
 		}
 	case admissionv1.Update:
+		req.DryRun = &defaultReqDryRun
```

Comment:
Please replace with `ptr.To(false)` and drop `defaultReqDryRun`

Reply:
Done, thx for the tip :)

Reply:
It failed to compile in the e2e test: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubernetes-sigs_cluster-api-provider-kubevirt/277/pull-kubernetes-sigs-cluster-api-provider-kubevirt-e2e/1764970340280176640

please run `go mod tidy`